packages/compass-aggregations/src/components/loading-overlay/loading-overlay.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import {
+  withTheme,
+  css,
+  cx,
+  SpinLoader,
+  spacing,
+  Body
+} from '@mongodb-js/compass-components';
+
+const loadingOverlayStyles = css({
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  bottom: 0,
+  right: 0,
+  zIndex: 1000,
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+});
+
+const loadingOverlayStylesDark = css({
+  backgroundColor: 'black',
+});
+
+const loadingOverlayStylesLight = css({
+  backgroundColor: 'white',
+});
+
+const textStyles = css({
+  marginLeft: spacing[1]
```

Comment:
```suggestion
  marginLeft: spacing[2]
```
Looked slightly better to me, how do you feel about it?